composer.json

```diff
@@ -15,14 +15,11 @@
   },
   "config": {
     "vendor-dir": "vendor",
-    "platform": {
-      "php": "5.5.9"
-    },
     "optimize-autoloader": true,
     "sort-packages": true
   },
   "require": {
-    "php": ">=5.5.9",
+    "php": "5.5.9 - 7.3",
```

Comment:
I think this would have some BC concerns. It's possible that some people may be running PHP 7.4, 8, etc, even though we don't officially support it yet. Aside from that, removing the platform reqs looks like a good idea.

Reply:
I had the same concerns. The platform constraint was the main point of this MR and i'm more than happy to remove this part to get it over the line.

I considered it worthwhile to define the upper limit as an aid to new installs.
If people are running >=7.4 then they hopefully have the knowledge to edit composer manually to overcome this. If they don't, then directing to 7.3 may be the best thing for everyone.

Reply:
I guess that the upper limit might be a good thing. If people are using 7.4, they have a real problem, many things will break, not just this. And I guess that if they're messing with **composer** they can tweak that value if they really need to. I'm not sure about this, "I'm just sayin..."